libminifi/include/processors/ListenHTTP.h

```diff
@@ -115,6 +114,9 @@ class ListenHTTP : public core::Processor {
   std::unique_ptr<Handler> _handler;
 };
 
+
+REGISTER_RESOURCE(ListenHTTP);
```

Comment:
While I am thinking of it, need to capture some of these details in docs for people that would like to contribute and give some indication of how the classloader functions.